src/main/java/org/fundacionjala/coding/fernando/Customer.java

```diff
@@ -0,0 +1,66 @@
+package org.fundacionjala.coding.fernando;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+
+class Customer {
+    private String name;
+    private ArrayList rentals = new ArrayList();
+
+    Customer(final String name) {
+        this.name = name;
+    }
+
+    public void addRental(final Rental arg) {
+        rentals.add(arg);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public double amountFor(final Rental rental) {
+        return rental.getCharge();
+    }
+
+    public String statement() {
+        Iterator iterator = this.rentals.iterator();
+        StringBuilder builder = new StringBuilder();
+        builder = builder.append("Rental Record for " + getName() + "\n");
+        while (iterator.hasNext()) {
+
+            Rental each = (Rental) iterator.next();
+            //show figures for this rental
+            builder = builder.append("\t" + each.getMovie().getTitle() + "\t"
```

Comment:
if you are using a StringBuilder, it's redundant to do this builder = builder.append
you should use only builder.append

and also it's redundant append using plus sign, append("\t" + 
all the values should concatenate using append